frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -21,73 +21,6 @@ if Device:hasFrontlight() then
     }
 end
 
-if Device:setDateTime() then
-    common_settings.time = {
-        text = _("Time and date"),
-        sub_item_table = {
```

Comment:
Has this been modified ? Or you just put it back way below where it was originally? (in which case, checkout the original, to limit the diff).